test/gtest/uct/ib/test_ib.cc

```diff
@@ -86,8 +86,21 @@ class test_uct_ib_addr : public test_uct_ib {
         uint16_t lid_out;
         enum ibv_mtu mtu;
         uint8_t gid_index;
+        size_t address_size;
+
+        address_size = uct_ib_iface_address_size(iface);
+
+        /* Hack: due to manipulation with subnet prefix in case if used custom
+         * subnet prefix value, size of packed address is calculated
+         * incorrectly: there is no room for subnet prefix value.
+         * Fix: just increase address size by missing room */
+        if (((subnet_prefix & UCT_IB_SITE_LOCAL_MASK) == UCT_IB_SITE_LOCAL_PREFIX)) {
+            address_size += sizeof(uint16_t);
+        } else if (subnet_prefix != UCT_IB_LINK_LOCAL_PREFIX) {
+            address_size += sizeof(uint64_t);
+        }
```

Comment:
can we use uct_ib_address_size() function instead, and pass custom params?

Reply:
yep, done